src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -4,22 +4,42 @@
 
 .hmrc-list-with-actions {
   @include govuk-font($size: 19);
+
+  @include govuk-media-query($from: desktop) {
+    .hmrc-summary-list__key {
```

Comment:
@timsb @kyle-bowden why can't we target the .govuk-summary-list__key here? I'm not sure I understand the reason for introducing a new class when these are scoped to .hmrc-list-with-actions

Reply:
My thoughts were that if `.govuk-summary-list__key` were to change it would either be more maintenance for us or could introduce a change that goes unnoticed (admittedly that's less likely with VRTs)

Reply:
so working it through, if govuk-summary-list markup was updated and someone saw that it was being used in this component then they might come and update it, but not realise that it was being used by css selector and then you could get a regression. That makes sense to me, and I can see how that could get past VRT tests, because person would already be expecting a change, so might not see that as well as the intended change there was a regression

Reply:
I wonder if we had it as hmrc-summary-list__key if we could maybe add an additional class to the container, alongside hmrc-list-with-actions we could have `hmrc-summary-list--without-value-column` so there's the old list with actions component, but also a more generic way that's just a hmrc variation of a summary summary list

Reply:
I generally prefer separating our own styling from any of the govuk styling to keep the concerns separate. `.hmrc-summary-list--without-value-column` would be the way to go about this but I'm not sure I like the fact that teams will have to add this when they don't want the value column. Would the logic for this be in play-frontend-hmrc? Also, would it make more sense for the logic to be the other way around i.e. `.hmrc-summary-list--with-value-column`?